src/coucoubot.ml

```diff
@@ -22,10 +23,10 @@ let config = {
   C.Config.
   server = "irc.freenode.net";
   port = 7000;
-  username = "coucoubot";
-  realname = "coucoubot";
-  nick = "coucoubot";
-  channel = "#arch-fr-free";
+  username = "roucoubot";
```

Comment:
heh, don't push your test setup here :laughing:

Reply:
Uh, thought I correctly add -p'd it :(